packages/manager/modules/billing-components/src/components/auto-renew-2016-deployment-banner/auto-renew-2016-deployment-banner.controller.js

```diff
@@ -0,0 +1,15 @@
+import { FAQ_LINK } from './auto-renew-2016-deployment-banner.constants';
+
+export default class AutoRenew2016DeploymentBannerController {
+  /* @ngInject */
+  constructor(coreConfig) {
+    this.coreConfig = coreConfig;
+  }
+
+  $onInit() {
+    this.FAQ_LINK =
+      FAQ_LINK[this.coreConfig.getUserLanguage().toUpperCase()] ||
+      FAQ_LINK[this.coreConfig.getUser().ovhSubsidiary] ||
```

Comment:
**nitpick:** Guides must always be displayed based on Subsidiary. Use only has the option to change the language in Manager.
```suggestion
      FAQ_LINK[this.coreConfig.getUser().ovhSubsidiary] ||
```

Reply:
I added subsidiaries as the guide links were only display based on language but I can remove the retrieve with user language as well